### DISCUSSION TITLE
Tradingview Webhook Bot Trading Platform

### DISCUSSION THREAD

**Discussion by bgoldmann2023** [FabCx/TradingView-Webhook-Bot#118] [General]
Looking to get this created cloud based can someone help me?

**Comment by carlostipton1319** [comment]
Additionally, do you have any preferences or requirements regarding the [send bitcoin](https://cryptonftbitcoin.com/how-to-send-bitcoin-from-paypal-to-another-wallet/ ) cloud platform or technologies to be used? The more information you can provide, the better I can assist you.

**Comment by fabston** [comment]
This isn’t the right place. Might wanna look at fiverr or freelancer.
